doc/developers/contributing.rst

```diff
@@ -207,6 +207,51 @@ and Cython optimizations.
    <http://astropy.readthedocs.org/en/latest/development/workflow/development_workflow.html>`_
    sections.
 
+Rebasing and squashing commits
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
```

Comment:
yes it gives [this](http://scikit-learn.org/stable/developers/contributing.html#instantiation) kind of a header.